### PR TITLE
Install and use pg-native.

### PR DESCRIPTION
**Pull request by caseycrites**
pg-native is a wrapper around the native bindings, which are supposed to be able to parse 20-30% faster, which is a Good Thing™.

Instructions pulled from https://github.com/brianc/node-postgres#native-bindings

### CODE REVIEW

**Comment by caseycrites** [comment]
I could be wrong about the order of operations here, but I'd guess the path to getting this where we need it is to upgrade here, point waterline-adapter-tests at it and update that repo to use pg-native as well, then point waterline at the new version of this, and THEN we can use it.

**Comment by kevinburkeshyp** [comment]
Waterline doesn't depend on sails-postgresql, we specify it as the adapter in the API

-kevin

**Comment by kevinburkeshyp** [comment]
But there's a circular dependency between this and the adapter tests

-kevin

File: lib/adapter.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ---------------------------------------------------------------*/
 
 // Dependencies
-var pg = require('pg');
+var pg = require('pg').native;
```
Comment (jterk) [inline lib/adapter.js:7]: It's a little annoying that this can't be done with the adapter config.

**Comment by jterk** [comment]
Looks OK to me - have you been able to verify use of the C lib? Does it still support the metrics I added today?

**Comment by caseycrites** [comment]
@jterk Good question. I'll spin something up when I get the chance and take a look.

**Comment by caseycrites** [comment]
@jterk I can confirm the native code being hit, and the metrics tests are still passing after merging master into this branch.

**Comment by jterk** [comment]
LGTM

**Pull request merged**